README.md

```diff
@@ -56,25 +56,6 @@ For more detailed instructions [see our wiki](https://github.com/theappbusiness/
 
 For more detailed information on how to setup your project and environment please see our [wiki](https://github.com/theappbusiness/MasterFastfile/wiki)
 
-## Provfiles
```

Comment:
The wiki needs some love before releasing the next version but it's probably a good idea to just ad some docs here on export option plists, and how to generate them.